src/dsp_tools/models/input_error.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+from typing import Protocol
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n\t"
+list_separator = "\n\t- "
+
+
+class Problem(Protocol):
+    """Information about input errors."""
+
+    def execute_error_protocol(self) -> str:
+        """
+        This function initiates all the steps for successful problem communication with the user.
+
+        Returns:
+            message for the error
+        """
+
+
+@dataclass(frozen=True)
+class DockerNotStartedProblem:
+    """This class contains information if the docker program is not started."""
+
+    user_msg: str
+
+
+@dataclass(frozen=True)
+class DirectoryProblem:
+    """This class contains information if a path does not exist."""
+
+    user_msg: str
```

Comment:
IMO this class should have a field `directory` which stores the non-existing directory. That might even make the message obsolete in this case?

Reply:
When I started this PR I though I would do the whole command, so those classes would be necessary. But as I only did the properties the problems would not arise here and they are therefore not implemented. Should I delete them in this PR?

Reply:
I don't mind having them here as placeholders